docs/community/topics/i18n.rst

```diff
@@ -125,26 +123,57 @@ Translate phrases, not words
 Full sentences and clauses must always be a single translatable string.
 Otherwise, you can get ``next page`` translated as ``suivant page`` instead of as ``page suivante``, etc.
 
-Deal with variables and markup in translations
+Dealing with variables and markup in translations
 `````````````````````````````````````````````````````````````
```

Comment:
```suggestion
Dealing with variables and markup in translations
`````````````````````````````````````````````````
```

OCD.

Reply:
Rahaaa. github does not understand, I reduced the number of backitcks to be the same length as the title.